courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>It seems that the model.fit method inspects the data, and if it's a generator will invoke automatically .fit_generator, so we don't really need to use it directly: .fit is enough even in the case we pass a tf.data.Dataset</p><p> </p><p>(It's here in the Keras code: <a href="https://github.com/keras-team/keras/blob/master/keras/engine/training.py#L1130-L1147" target="_blank">https://github.com/keras-team/keras/blob/master/keras/engine/training.py#L1130-L1147</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
ok. interesting. I was getting an odd error when I used just .fit. Googling/searching didn't help make must sense of it. I'll show you in our sync.